Napzak-iOS/Napzak-iOS/Presentation/Search/View/SearchView.swift

```diff
@@ -146,7 +146,11 @@ extension SearchView {
         LazyVStack(spacing: 0) {
             ForEach(viewModel.productFetchOption.genres.indices, id: \.self) { i in
                 Button {
-                    //TODO: - 장르 페이지로 이동
+                    let id = viewModel.productFetchOption.genres[i].id
+                    let name = viewModel.productFetchOption.genres[i].name
```

Comment:
```suggestion
let genre = viewModel.productFetchOption.genres[i]
let id = genre.id
let name = genre.name
```
저느 중복되는 접근이니깐 변수로 빼서 할 것 같네요